circle.yml

```diff
@@ -25,24 +21,17 @@ machine:
 dependencies:
   cache_directories:
     - "node_modules"
-    - "~/.npm/_git-remotes/"
   pre:
-      #enable xdebug. LINE 1/2 to uncomment if you want to run a code coverage report.
-      - sed -i 's/^;//' ~/.phpenv/versions/$(phpenv global)/etc/conf.d/xdebug.ini
       #setup WP install
       - git clone git://develop.git.wordpress.org/ $WP_CORE_DIR;
       - cd $WP_CORE_DIR && cp wp-tests-config-sample.php wp-tests-config.php && sed -i "s/youremptytestdbnamehere/wordpress_test/" wp-tests-config.php && sed -i "s/yourusernamehere/root/" wp-tests-config.php && sed -i "s/yourpasswordhere//" wp-tests-config.php;
       # set up database
       - mysql -e 'CREATE DATABASE wordpress_test;' -uroot;
-      # setup phpunit
-      - wget https://phar.phpunit.de/phpunit-old.phar && chmod +x phpunit-old.phar && mv phpunit-old.phar /home/ubuntu/.phpenv/shims/phpunit
-      #Sensei Specific remove npm file as it is not needed
       # Clone sensei-coding-standard
       - git clone https://github.com/Automattic/sensei-coding-standard.git $SENSEI_CODING_STANDARD_DIR;
 
 ## tests override
 test:
   override:
   #run tests with coverage
-  - cd $SENSEI_CODING_STANDARD_DIR; php bin/phpcs.phar --report-full --report-summary --standard=Sensei $plugin_loc
-  - cd $plugin_loc; phpunit #ignoring coverage for speed reasons: --coverage-html $CIRCLE_ARTIFACTS
```

Comment:
Isn't this the line that runs the phpunit test suite? If so, I don't think we should remove it.

Reply:
CCI calls it implicitly. We don't need it as per https://circleci.com/docs/1.0/language-php/#testing